test/DocumentFormat.OpenXml.Packaging.Tests/PartConstraintRuleTests.cs

```diff
@@ -50,6 +50,20 @@ public void AllExpectedParts()
                 .OrderBy(v => v, StringComparer.Ordinal)
                 .ToList();
 
+#if DEBUG
+            var tmp = Path.GetTempFileName();
```

Comment:
This seems to be duplicated a few times. Is this to help with generating the parts tests?

Could you consolidate any of this to a centralized extension method or something?

Reply:
Yes, it's for the part constraint testing updates after adding parts. And yes, I probably can turn this into an extension so I'll add something to this commit.